backend/src/main/resources/schema.sql

```diff
@@ -0,0 +1,33 @@
+CREATE TABLE STAFF (
+  id INT AUTO_INCREMENT  PRIMARY KEY,
+  first_name VARCHAR(255) NOT NULL,
+  surname VARCHAR(255) NOT NULL,
+  active BOOLEAN NOT NULL DEFAULT TRUE,
+  start_date DATE,
+  contracted_hours DECIMAL,
+  hourly_rate DECIMAL,
+  role ENUM('user', 'manager') NOT NULL,
+  job_title VARCHAR(255)
+);
+
+CREATE TABLE PREFERRED_DATES (
+    staff_id INT REFERENCES STAFF(id) ON DELETE CASCADE,
+    PRIMARY KEY(staff_id),
+    monday BOOLEAN,
+    tuesday BOOLEAN,
+    wednesday BOOLEAN,
+    thursday BOOLEAN,
+    friday BOOLEAN,
+    saturday BOOLEAN,
+    sunday BOOLEAN
+);
```

Comment:
I think this would be better modelled for the now as a array of VARCHARs. It can still be modelled as a Enum in the java code and thrown into the database using the the Enums toString method. https://www.baeldung.com/jpa-persisting-enums-in-jpa Helps though admittedly it is using JPA instead of JBDC.

Reply:
Not sure how to create an array of varchar in sql (or an array of anything without creating another table).
Writing a String[] in staff.java and having jpa handle it results in varbinary(255), which is not something we can set easily with data.sql. It also generates spotbugs security warnings.